crates/hyperqueue/src/client/task.rs

```diff
@@ -32,7 +36,19 @@ pub struct TaskListOpts {
     pub verbosity: VerbosityFlag,
 }
 
-pub async fn output_job_tasks(
+#[derive(clap::Parser)]
+pub struct TaskInfoOpts {
+    /// Select specific job(s).
+    pub job_selector: JobSelectorArg,
```

Comment:
We assume a single job here, right? So this should not allow passing multiple job IDs.

Reply:
Yes, you're right.
This PR should serve as a base for additional (#458) issues. Now it just solves a problem with a big task list output by moving some of the info into task info. Also the usage would be similar as is for a job (job list, job info).